Overview.src.html

```diff
@@ -622,34 +631,42 @@ <h4 id=terminology-headers>Headers</h4>
    <td>"<code title>document</code>"
    <td>HTML's navigate algorithm.
   <tr>
+   <td>"<code title>document</code>"
+   <td><code title>child-src</code>
+   <td>HTML's <code>&lt;iframe></code> and <code>&lt;frame></code>
+  <tr>
    <td>"<code title>subresource</code>"
    <td><code title>connect-src</code>
    <td><code>navigator.sendBeacon()</code>, <code>EventSource</code>,
```

Comment:
That sounds good. I'm having a hard time judging whether a report is a type or a destination, but I think this makes sense.